src/foam/u2/view/LoginView.js

```diff
@@ -110,6 +110,9 @@ foam.CLASS({
   ^ .align-end {
     text-align: end;
   }
+  ^ .margin-top {
+    margin-top: 16px;
+  }
```

Comment:
Any problem with always having this css class enabled?

Reply:
there is an extra gap 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/95241308/230104421-b0485de0-4cf8-4646-a261-e2f32956dc0e.png">